test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/CIEnvironmentVariableTests.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using Datadog.Trace.Ci;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI
+{
+    [CollectionDefinition(nameof(CIEnvironmentVariableTests), DisableParallelization = true)]
+    public class CIEnvironmentVariableTests
+    {
+        private IDictionary _originalEnvVars;
+
+        public CIEnvironmentVariableTests()
+        {
+            _originalEnvVars = Environment.GetEnvironmentVariables();
+        }
+
+        public static IEnumerable<object[]> GetJsonItems()
+        {
+            // Check if the CI\Data folder exists.
+            string jsonFolder = Path.Combine("CI", "Data");
+            if (!Directory.Exists(jsonFolder))
+            {
+                throw new DirectoryNotFoundException(jsonFolder);
+            }
+
+            // JSON file path
+            foreach (string filePath in Directory.EnumerateFiles(jsonFolder, "*.json", SearchOption.TopDirectoryOnly))
```

Comment:
Is there a reason that you're pre-translating the original yaml file to JSON? Would it be easier to copy over the yaml into this repo unmodified, and then use a yaml-parsing library to convert it into a dictionary? If the spec updates, I'm thinking this could save time over the long run

Reply:
Well I’m doing pretty much what other tracers does. You can see in the makefile: https://github.com/DataDog/datadog-ci-spec/blob/main/Makefile

When the spec is updated a github action is triggered downloading the repo and replacing the jsons.

Reply:
https://github.com/DataDog/datadog-ci-spec/actions/runs/502805998

Reply:
Thanks for clarifying! 😄 I wanted to minimize the manual steps needed to update our specs/tests and it looks like that's taken care of 👍🏼